client/state/selectors/is-site-google-my-business-eligible.js

```diff
@@ -45,22 +45,42 @@ export const siteHasBusinessPlan = createSelector(
 	( state, siteId ) => [ getSitePlanSlug( state, siteId ) ]
 );
 
+/**
+ * Returns true if site has jetpack premium/business plan
+ *
+ * @param  {Object}  state  Global state tree
+ * @param  {String}  siteId The Site ID
+ * @return {Boolean} True if site has business plan
+ */
+export const siteHasEligibleJetpackPlan = createSelector(
+	( state, siteId ) => {
+		const slug = getSitePlanSlug( state, siteId );
+
+		return (
+			planMatches( slug, { group: GROUP_JETPACK, type: TYPE_PREMIUM } ) ||
+			planMatches( slug, { group: GROUP_JETPACK, type: TYPE_BUSINESS } )
+		);
```

Comment:
@yurynix: why did we not use [`isJetpackBusinessPlan()` and `isJetpackPremiumPlan()` methods](https://github.com/Automattic/wp-calypso/blob/e1122c1fb71525ea1720b4f3e9b6a89045cebca3/client/lib/plans/index.js#L226)?

Reply:
Because that what `isSiteGoogleMyBusinessEligible` were using.

Reply:
I'm not sure I understand your comment. Could you clarify it for me? I was suggesting the following alternative code:

```javascript
return isJetpackBusinessPlan( slug ) && isJetpackPremiumPlan( slug );
```

Reply:
The other function in that file, `isSiteGoogleMyBusinessEligible`, uses the same method to get the plan, so that what I used. You're welcome to refactor both functions to whatever you think is best.